packages/dashboard/src/components/widgets/tile/tile.tsx

```diff
@@ -100,10 +99,12 @@ const WidgetTile: React.FC<WidgetTileProps> = ({ children, widget, title, remove
             borderBottom: `2px solid ${colorBorderDividerDefault}`,
           }}
         >
-          <Box variant='h1' fontSize='body-m'>
-            {title}
-          </Box>
-          <SpaceBetween size='s' direction='horizontal'>
+          <div className='widget-tile-header' title={title}>
+            <Box variant='h1' fontSize='body-m'>
+              {title}
+            </Box>
+          </div>{' '}
+          <div style={{ display: 'flex' }}>
```

Comment:
please move this to a CSS file

Reply:
I moved it to css file, please review it.